test/system/comment_test.rb

```diff
@@ -80,8 +85,47 @@ def setup
       find("p", text: comment_response_text)
     end
 
+    test "post #{page_type_string}, then comment on FRESH #{page_type_string}" do
+      title_text, body_text = String.new, String.new
+      case page_type_string
```

Comment:
Wow, this is really cool !!
How is `page_type_string ` getting substituted?

Reply:
It's happening here: https://github.com/publiclab/plots2/blob/9fb5d65ffa744623064a5c6353618c01c6c33ea4/test/test_helper.rb#L43-L50

Helper function that returns the name of the node fixture :)